numba/core/typing/templates.py

```diff
@@ -999,15 +999,17 @@ def get_template_info(self):
         return info
 
 
-def make_intrinsic_template(handle, defn, name, kwargs):
+def make_intrinsic_template(handle, defn, name, *, prefer_literal=False,
+                            kwargs={}):
```

Comment:
```suggestion
                            kwargs=None):
   kwargs = MappingProxyType({} if kwargs is None else kwargs)
```
should this be like this to avoid adding a mutable kwarg default?

Reply:
Good point